### PR TITLE
Non-bot messages will now be deleted in the suggestions channel.

### LINKED ISSUE #7: Messages do not get deleted in #suggestions
### Explain the issue.

Old bot impl was that non-bot messages get deleted in suggestions channel. This has disappeared!

### How to recreate?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Zcamt**
This will bring back the old implementation where non-bot messages are deleted as outlined in #7

### CODE REVIEW

**Review by Nuckerr — CHANGES_REQUESTED** [review]
Would it maybe be worth putting this in another file. Its not the most intuitive place to put this. Either put it in `LearnspigotBot` or make its own file in a `listener` package

**Comment by Zcamt** [comment]
> Would it maybe be worth putting this in another file. Its not the most intuitive place to put this. Either put it in `LearnspigotBot` or make its own file in a `listener` package

Did think about that but wasn't too sure, I think having a dedicated listener package might be the best call though as this'll also allow for similar additions in the future, will update in a bit 👍

**Review by Nuckerr — APPROVED** [review]
Will merge after workflows branch is merged

**Pull request merged**
